lib/bugsnag/api/client/errors.rb

```diff
@@ -42,7 +42,15 @@ def update_errors(project_id, ids, operation, options = {})
           when String
             patch "projects/#{project_id}/errors/#{ids}", options.merge({:operation => operation})
           when Array
-            patch "projects/#{project_id}/errors", options.merge({:operation => operation, :query => {:error_ids => ids.join(' ')}})
+            defaults = {:operation => operation, :query => {:error_ids => ids.join(' ')}}
+            merged_opts = defaults.merge(options) do |key, oldVal, newVal|
```

Comment:
For consistency, use snake_case for symbols, methods and variables.

Reply:
You could add rubocop to the project

Reply:
I'll fix this for now but add a ticket for rubocop.  781 issues running locally, so will need some fixing.